juju/juju
# Removes deprecated update-series jujud command.

## Description of change

This patch removes the deprecated _jujud_ command, `update-series`.

No dependencies are removed; those in the system package are recruited by the superseding upgrade-series worker et al.

## QA steps

- Bootstrap and add a Trusty machine, and SSH to it.
- Run `juju-update-series --from-series trusty --to-series xenial`. The command should not be found.

## Documentation changes

None.

## Bug reference

N/A
